packages/object-utils/package.json

```diff
@@ -18,14 +18,14 @@
     "node": ">=18"
   },
   "scripts": {
-    "build": "vite build && pnpm types:build && tsc-alias -p tsconfig.build.json",
+    "build": "scalar-build-rollup",
```

Comment:
We should consider to empty the dist folder in `scalar-build-rollup`. Keeping old files can be confusing when  testing locally.